Meetings/Meeting.ics

```diff
@@ -1,67 +1,150 @@
 BEGIN:VCALENDAR
 PRODID:-//Microsoft Corporation//Outlook 16.0 MIMEDIR//EN
 VERSION:2.0
-METHOD:PUBLISH
+METHOD:REQUEST
 X-MS-OLK-FORCEINSPECTOROPEN:TRUE
 BEGIN:VTIMEZONE
-TZID:UTC
+TZID:Central Standard Time
 BEGIN:STANDARD
-DTSTART:16010101T000000
-TZOFFSETFROM:-0000
-TZOFFSETTO:-0000
+DTSTART:16011104T020000
+RRULE:FREQ=YEARLY;BYDAY=1SU;BYMONTH=11
+TZOFFSETFROM:-0500
+TZOFFSETTO:-0600
 END:STANDARD
+BEGIN:DAYLIGHT
+DTSTART:16010311T020000
+RRULE:FREQ=YEARLY;BYDAY=2SU;BYMONTH=3
+TZOFFSETFROM:-0600
+TZOFFSETTO:-0500
+END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VEVENT
+ATTENDEE;CN="Madison Hardy";RSVP=TRUE:invalid:nomail
+ATTENDEE;CN="Rob Prouse (.NET Foundation)";RSVP=TRUE:invalid:nomail
+ATTENDEE;CN="Mattias Karlsson (.NET Foundation)";RSVP=TRUE:invalid:nomail
+ATTENDEE;CN="Tom Pappas (.NET Foundation)";ROLE=OPT-PARTICIPANT;RSVP=TRUE:i
+	nvalid:nomail
+ATTENDEE;CN="Madison Hardy";ROLE=OPT-PARTICIPANT;RSVP=TRUE:mailto:MHardy@vi
+	rtualinc.com
```

Comment:
Would it just be better to not list anyone? Maybe that's why Outlook is getting confused?